tests/impl.cpp

```diff
@@ -94,10 +118,14 @@ class SSE2NEONTestImpl : public SSE2NEONTest
                 // Make sure the NaN values are included in the testing
                 // one out of four times.
                 if ((rand() & 3) == 0) {
+                    // FIXME:
+                    // The argument "0xFFFFFFFFFFFF" is a tricky workaround to
```

Comment:
Can you avoid using string literal "0xFFFFFFFFFFFF" ?

Reply:
Any suggestion?
I would like to leave the whole `FIXME` comment and fix it in the future pull request.
To avoid using `0xFFFFFFFFFFFF` in `nanf()`, the testing code structure should be rewritten.